build-tools/packages/build-cli/src/commands/report/codeCoverage.ts

```diff
@@ -56,12 +57,12 @@ export default class ReportCodeCoverageCommand extends BaseCommand<
 		codeCoverageAnalysisArtifactNameBaseline: Flags.string({
 			description: "Code coverage artifact name for the baseline build.",
 			env: "CODE_COVERAGE_ANALYSIS_ARTIFACT_NAME_BASELINE",
-			required: true,
+			required: false,
```

Comment:
This seems related to my comment from the other PR, right? If so, any reason to not just remove the flags altogether? I think a `const artifactNamePrefix = "Code Coverage Report";` in this file would be ok.

Reply:
ok.